elo_loyalty_prediction.ipynb

```diff
@@ -87,7 +86,18 @@
    "metadata": {},
    "outputs": [],
    "source": [
-    "hist_trans_df.head()"
+    "train_and_validation_df.head()"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Create Features\n",
+    "\n",
+    "Next we want to combine and shape all of our raw data to create useful features in the train data set.\n",
+    "\n",
+    "For now, I'll do this step-by-step here to make it a bit easier to understand (for myself), but later we'll probably want to do this in some python classes, or this file will be huge. // Erich"
```

Comment:
It sounds good also for me, so I can for now just try to follow up what you are doing and I have everything in one place.